browser/ui/BUILD.gn

```diff
@@ -235,6 +235,8 @@ source_set("ui") {
     "//brave/components/brave_wayback_machine:buildflags",
     "//brave/components/cosmetic_filters/resources/data:generated_resources",
     "//brave/components/crypto_dot_com/browser/buildflags:buildflags",
+    "//brave/components/ftx/browser/buildflags:buildflags",
+    "//brave/components/ftx/common",
```

Comment:
this needs to be inside enable_ftx to match https://github.com/brave/brave-core/pull/8629/files#diff-7deaca579ce61106651458308d2792dc2e5bf9f6958adeb897cac92c45263f24R54

Reply:
fixed via https://github.com/brave/brave-core/compare/6f4c503619e82a440ef7db9782288e922c37d5e4..f4d887c0baf39543331107f2e22f9b0ad4986c02